credits.txt

```diff
@@ -11,6 +11,11 @@ Discord (invite: ZeuASSx)
 
 Programming
```

Comment:
Might be worth renaming this to "Creator".

Reply:
Changed. Thanks!